assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -61,13 +69,41 @@ class Draggable extends Component {
 	 * @param {Object} event The non-custom DragEvent.
 	 */
 	onDragEnd = ( event ) => {
-		const { onDragEnd = noop } = this.props;
+		const { clearHighlight, dropElementByOffset, blockName, setTimeout, onDragEnd = noop } = this.props;
 		if ( event ) {
 			event.preventDefault();
 		}
 
+		// Make sure to clear highlight.
+		clearHighlight();
+
+		// Attempt drop on neighbor if offset
+		if ( this.pageOffset !== 0 ) {
+			// All this is about calculating the position of the (correct) element on the new page.
+			const currentElementTop = parseInt( this.cloneWrapper.style.top );
+			const currentElementLeft = parseInt( this.cloneWrapper.style.left );
+			const newLeft = currentElementLeft - ( this.pageOffset * PAGE_AND_MARGIN );
+
+			let baseHeight, xAttribute, yAttribute;
+			if ( isCTABlock( blockName ) ) {
+				baseHeight = STORY_PAGE_INNER_HEIGHT / 5;
```

Comment:
What is the / 5 do here. If this should it's own constant on this line needs to be documented.

Reply:
Agreed.

The same logic is already used a bit below:

```js
// 20% of the full value in case of CTA block.
const baseHeight = blockIsCTA ? STORY_PAGE_INNER_HEIGHT / 5 : STORY_PAGE_INNER_HEIGHT;
```

So extracting it makes sense.

The reason is that the CTA layer in an AMP story can only be placed i n the lower 20% of the page, hence the division by 5.

Reply:
So if this logic is repeated then it should be a const for sure. It will make it easier refactor later. 

So maybe `STORY_PAGE_INNER_HEIGHT_FOR_CTA` as a name?

Reply:
Actual placement of the element is done in CSS:

https://github.com/ampproject/amp-wp/blob/develop/assets/src/stories-editor/blocks/amp-story-cta/edit.css#L1-L6

I have moved this to a new constant in 005bac0.